external-deps/spyder-kernels/spyder_kernels/_version.py

```diff
@@ -8,5 +8,5 @@
 
 """Version File."""
 
-VERSION_INFO = (1, 10, 0, 'dev0')
+VERSION_INFO = (2, 0, 0, 'dev0')
```

Comment:
@impact27, you incorrectly cloned master of spyder-kernels against our 4.x branch here. Please fix that.

Reply:
done